docs/standard/microservices-architecture/multi-container-microservice-net-applications/subscribe-events.md

```diff
@@ -13,12 +13,19 @@ ms.topic: article
 
 The first step for using the event bus is to subscribe the microservices to the events they want to receive. That should be done in the receiver microservices.
 
-The following simple code shows what each receiver microservice needs to implement when starting the service (that is, in the Startup class) so it subscribes to the events it needs. For instance, the basket.api microservice needs to subscribe to ProductPriceChangedIntegrationEvent messages. This makes the microservice aware of any changes to the product price and lets it warn the user about the change if that product is in the user’s basket.
+The following simple code shows what each receiver microservice needs to implement when starting the service (that is, in the `Startu`p class) so it subscribes to the events it needs. In this case, the `basket.api` microservice needs to subscribe to `ProductPriceChangedIntegrationEvent` and the `OrderStartedIntegrationEvent` messages. 
```

Comment:
Closing back tick should be after the 'p' in `Startup`.

Reply:
👍